beta-src/src/utils/api/index.ts

```diff
@@ -41,9 +47,9 @@ export const submitOrders = (
 ): Promise<AxiosResponse> => {
   if (Object.keys(queryParams).length) {
     return orderSubmission.post(
-      `ajax.php?${buildQueryString(queryParams)}`,
+      `../ajax.php?${buildQueryString(queryParams)}`,
```

Comment:
Using relative API calls was in support of getting `npm start` to work... Because then we should be able to use `proxy` in package.json to proxy out these requests to the docker backend.

Anyway, this doesn't really belong in this diff... it's covered in https://github.com/codazen/webDiplomacy/pull/129 . If anyone can figure out how to get that working I will be indebted!

Reply:
@neillabsai Could you help here?